code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -92,6 +92,13 @@
 /turf/open/return_analyzable_air()
 	return return_air()
 
+/turf/open/proc/has_gas(datum/gas, mole_amount)
+	if(air && air.total_moles())
```

Comment:
total_moles() is not relevant here

Reply:
So the reason for using `total_moles()` here is due to the possibility of space turfs.  They have an air variable but it creates a runtime since it's different from regular turfs.  The total moles is a cheap hack to check and see if there is any air pressure.

I used this as my reference. 

https://github.com/tgstation/tgstation/blob/2a90fdd75e1488c4be03cdfcce20b70de35a173d/code/modules/mob/living/carbon/human/species_types/plasmamen.dm#L49

Reply:
oh for god's sake you're right

Reply:
73324af avoids using `total_moles()` since the `has_gas` proc uses `ASSERT_GAS` which creates a temporary gas on the space turf to check if there is gas.  Then later it gets deleted by the GC